conans/client/remote_manager.py

```diff
@@ -36,7 +36,14 @@ def upload_conan(self, conan_reference, remote):
         the_files = compress_conan_files(the_files, export_folder, EXPORT_TGZ_NAME,
                                          CONANFILE, self._output)
 
-        return self._call_remote(remote, "upload_conan", conan_reference, the_files)
+        ret = self._call_remote(remote, "upload_conan", conan_reference, the_files)
+        msg = "Uploaded conan recipe '%s' to '%s'" % (str(conan_reference), remote.name)
+        # FIXME: server dependent
+        if remote.url == "https://server.conan.io":
```

Comment:
It won't harm, especially if you are managing multiple remotes, seeing a msg with the URL of the one you just uploaded to can be nice.